src/main/java/com/napier/sem/Record.java

```diff
@@ -0,0 +1,147 @@
+package com.napier.sem;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+/**
+ * Class representing a database record which could be one of:
+ *   - Country
+ *   - City
+ *   - Capital City
+ * The possible fields consist of:
+ *   code, name, continent, region, country,
+ *   district, capital, population
```

Comment:
Do you want to add recordType here? Not really a field but thought it was worth checking anyway.

Reply:
probably not, but we should be mentioning it in the comment i think. i'll update.